crux-mir/src/Mir/Trans.hs

```diff
@@ -323,7 +324,7 @@ extendToMax n e1 m e2 (Just arith) k =
                           Just LeqProof ->
                               k n e1 (S.app $ extend n m e2)
                           Nothing -> error "impossible case"
-extendToMax n e1 m e2 Nothing k = 
+extendToMax n e1 m e2 Nothing k =
```

Comment:
Just being picky, but I usually try to make unrelated changes line this in different PR's.

Reply:
There were trailing spaces on some of the lines I was editing, so I just ran `delete-trailing-spaces` on the whole file.

Reply:
I love ws-butler-mode (in emacs): it automatically removes extraneous whitespace around the areas you are editing, but leaves the rest of the file alone.  This obviously doesn't fully convert the entire file, but it does avoid introducing these un-related changes into your current PR whilst ensuring the portions you've worked on are in good shape.